frontend/src/components/Calendar/CalendarHeader.js

```diff
@@ -1,37 +1,95 @@
-    return (
-        <header className={`d-flex align-items-center ${styles.header}`}>
-            <div className={styles.buttonContainer} >
-                <div className={`${styles.icon_sentence} d-flex align-items-center mt-4`} onClick={() => dispatch(toggle())} >
-                    <img src={icon} alt="Add Icon" className={styles.icon} />
-                    <span className={styles.sentence}>Create</span>
-                </div>
-                <button className={styles.button} onClick={() => { dispatch(reset()) }}>Today</button>
-            </div>
-            <button onClick={() => { dispatch(decrease()) }} className={styles.buttonArrow}>
-                <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" className={" bi bi-chevron-left"} viewBox="0 0 16 16">
-                    <path fillRule="evenodd" d="M11.354 1.646a.5.5 0 0 1 0 .708L5.707 8l5.647 5.646a.5.5 0 0 1-.708.708l-6-6a.5.5 0 0 1 0-.708l6-6a.5.5 0 0 1 .708 0z" />
-                </svg>
-            </button>
+  const MonthIndex = useSelector(state => state.MonthIndex.value);
+  const [showInviationBox, setShowInvitationBox] = useState(false);
+  const dispatch = useDispatch(); //dispatch is touse function to interact with State of Redux
+  const handleInvitationBox = () => {
+    setShowInvitationBox(!showInviationBox);
+  };
+  return (
+    <header className={`d-flex align-items-center ${styles.header}`}>
+      <div className={styles.buttonContainer}>
+        <div
+          className={`${styles.icon_sentence} d-flex align-items-center mt-4`}
+          onClick={() => dispatch(toggle())}
+        >
+          <img src={icon} alt="Add Icon" className={styles.icon} />
+          <span className={styles.sentence}>Create</span>
+        </div>
+        <div onClick={handleInvitationBox}>Show Invitation Box</div>
+        <button onClick={handleInvitationBox}>Unshow Invitation Box</button>
+        {/*we can add icon then */}
+        <button
+          className={styles.button}
+          onClick={() => {
+            dispatch(reset());
+          }}
+        >
+          Today
+        </button>
+        <button></button>
+      </div>
+      <div>
+        {showInviationBox && (
+          <div className={styles.invitationBox}>
+            <InviationModal />
+          </div>
+        )}
+      </div>
+      <button
+        onClick={() => {
+          dispatch(decrease());
+        }}
+        className={styles.buttonArrow}
+      >
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          width="16"
+          height="16"
+          fill="currentColor"
+          className={' bi bi-chevron-left'}
+          viewBox="0 0 16 16"
+        >
+          <path
+            fillRule="evenodd"
+            d="M11.354 1.646a.5.5 0 0 1 0 .708L5.707 8l5.647 5.646a.5.5 0 0 1-.708.708l-6-6a.5.5 0 0 1 0-.708l6-6a.5.5 0 0 1 .708 0z"
+          />
+        </svg>

```

Comment:
can we make this a separate component?

Reply:
Of course! I just put all small icons into one icons.js file in frontend/src/assets